frog/imports/ui/Preview/Preview.js

```diff
@@ -310,6 +310,7 @@ export const StatelessPreview = withState(
                     config={activityData.config}
                     reload={reload}
                     doc={dashboard}
+                    instances={users.map((u, i) => Math.ceil(i / 2))}
```

Comment:
I think that is not correct, in that list instances will appear as many times as they have users.